### ISSUE TITLE
Selenide 6.8.0 seems to exclude io.opentelemetry also from seleniumhq dependencies

### ISSUE THREAD

**Issue by juanpablo-santos** [selenide/selenide#1965]
## The problem
After upgrading to Selenide 6.8.0 we're facing an `java.lang.NoClassDefFoundError: io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdk` exception when doing `new RemoteWebDriver( hubURL, chromeOptions );`. 

## Details
Upon looking dependency tree, after upgrading to Selenide 6.8.0, the `io.opentelemetry` jars are missing, seems caused by #1963.

## Tell us about your environment
* **Selenide Version**: 6.8.0
* **Chrome\Firefox\IE Version**: 
* **Browser Driver Version**: 
* **Selenium Version**: 
* **OS Version**: 

## Code To Reproduce Issue [ Good To Have ]
On a simple maven project add just the selenide dependency like:
```
    <dependency>
      <groupId>com.codeborne</groupId>
      <artifactId>selenide</artifactId>
      <scope>compile</scope>
      <version>6.8.0</version>
    </dependency>
```

Running `mvn dependency:tree  -Dincludes=io.opentelemetry` yields 
```
[INFO] -----------------------< io.corp.example:example >-----------------------
[INFO] Building example 1.0.0
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ stats ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.090 s
[INFO] Finished at: 2022-09-26T11:18:16+02:00
[INFO] ------------------------------------------------------------------------
```

Adding a second dependency so that it now looks like:
```
    <dependency>
      <groupId>com.codeborne</groupId>
      <artifactId>selenide</artifactId>
      <scope>compile</scope>
      <version>6.8.0</version>
    </dependency>

    <dependency>
      <groupId>org.seleniumhq.selenium</groupId>
      <artifactId>selenium-remote-driver</artifactId>
      <scope>compile</scope>
      <version>4.4.0</version>
    </dependency>
```

then running `mvn dependency:tree  -Dincludes=io.opentelemetry` this time yields 
```
[INFO] -----------------------< io.corp.example:example >-----------------------
[INFO] Building example 1.0.0
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ stats ---
[INFO] io.corp.example:example:jar:1.0.0
[INFO] \- org.seleniumhq.selenium:selenium-remote-driver:jar:4.4.0:compile
[INFO]    +- io.opentelemetry:opentelemetry-api:jar:1.16.0:compile
[INFO]    +- io.opentelemetry:opentelemetry-context:jar:1.16.0:compile
[INFO]    +- io.opentelemetry:opentelemetry-exporter-logging:jar:1.16.0:compile
[INFO]    |  +- io.opentelemetry:opentelemetry-sdk-metrics:jar:1.16.0:compile
[INFO]    |  \- io.opentelemetry:opentelemetry-sdk-logs:jar:1.16.0-alpha:compile
[INFO]    +- io.opentelemetry:opentelemetry-sdk-common:jar:1.16.0:compile
[INFO]    +- io.opentelemetry:opentelemetry-sdk-extension-autoconfigure-spi:jar:1.16.0:compile
[INFO]    +- io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:jar:1.16.0-alpha:compile
[INFO]    +- io.opentelemetry:opentelemetry-sdk-trace:jar:1.16.0:compile
[INFO]    +- io.opentelemetry:opentelemetry-sdk:jar:1.16.0:compile
[INFO]    \- io.opentelemetry:opentelemetry-semconv:jar:1.16.0-alpha:compile
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.124 s
[INFO] Finished at: 2022-09-26T11:20:01+02:00
[INFO] ------------------------------------------------------------------------
```

The code highlighting the issue is just a simple `new RemoteWebDriver( hubURL, chromeOptions );`, with the exception stacktrace looking like:
```
java.lang.NoClassDefFoundError: io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdk

	at org.openqa.selenium.remote.tracing.opentelemetry.OpenTelemetryTracer.createTracer(OpenTelemetryTracer.java:78)
	at org.openqa.selenium.remote.tracing.opentelemetry.OpenTelemetryTracer.getInstance(OpenTelemetryTracer.java:60)
	at org.openqa.selenium.remote.RemoteWebDriver.createExecutor(RemoteWebDriver.java:180)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:139)
[...]
```

(clarification edit: we do that `webDriver = new RemoteWebDriver(..)` so we can afterwards call `com.codeborne.selenide.WebDriverRunner.setWebDriver( webDriver );` )

**Comment by BorisOsipov** [comment]
Workaround: 
new RemoteWebDriver( hubURL, chromeOptions, **false** );

**Comment by juanpablo-santos** [comment]
> Workaround: 
new RemoteWebDriver( hubURL, chromeOptions, **false** );

indeed, thanks for your fast response!

**Comment by asolntsev** [comment]
@juanpablo-santos Yes, we explicitly excluded `opentelemetry` dependency because we assumed nobody really uses it. 
Probably it was a bad idea, will restore `opentelemetry` dependency in 5.8.1. 

P.S. In most cases it's not needed to call `new RemoteWebDriver(..)` and `WebDriverRunner.setWebDriver(..)`. There are better ways. See https://selenide.org/2019/12/03/advent-calendar-set-webdriver-vs-webdriver-provider/

**Comment by juanpablo-santos** [comment]
Hi @asolntsev ,

my understading of #1963 was that opentelemetry was removed as a _direct_ dependency of Selenide, as it wasn't used in the code, not that it should have been also removed as a _transitive_ dependency, so the exception surprised me and I opened the issue. But that was my understanding. 

We aren't using `opentelemetry`, the code that hitted the exception was rather old, and both the workaround and the link provided are super helpful, so there's no need for us to bring it back, the fewer dependencies the better :-) 

thanks for your suggestions and your prompt response!
